website/versioned_docs/version-23.x/Puppeteer.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: version-23.x-puppeteer
+id: version-24.x-puppeteer
```

Comment:
says 24, but sits in 23. Also, looks like this could be moved entirely to 22 anyway

Reply:
not 22 as this uses a preset that doesn't work with 22 (I think).

Anyways, when we release 27 I'll delete v22

Reply:
hah, didn't fact-check that. Deleting 22 for 27 sounds fair 👍